examples/plot_hf_hub.py

```diff
@@ -162,6 +162,7 @@
 
 hub_utils.update_env(path=local_repo, requirements=["scikit-learn"])
 
+
```

Comment:
Why this change? We don't consistently use 2 empty lines.

Reply:
I addressed this